pages/blog/announcing-graphql-yoga-2.mdx

```diff
@@ -0,0 +1,278 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'Announcing GraphQL Yoga 2.0!',
+  tags: ['server', 'envelop', 'node', 'graphql'],
+  author: 'guild',
+  date: '2022-03-29',
+  description:
+    'Fully-featured GraphQL Server with focus on easy setup, performance and great developer experience',
+  image: '/blog-assets/angular-cli-meteor-no-eject/cover.png'
+}
+
+export default withArticle({ ...meta })
+
```

Comment:
Should we start with embedding here @notrab 's video on Yoga?

Reply:
In the "Getting Started" section, yes!

Reply:
I’m happy to update the intro and re-release, but I couldn’t do it until Sunday/Monday morning.

Reply:
we can use the former one for Monday and replace later, I think it was still really good

Reply:
@Urigo I'm happy with that 🚀

Reply:
> Not in the very beginning?
> Video is always easier for people to watch no?

Having the video at the very beginning of the article would feel like the article is just a summary of the video, which is not the case 👀 

We have a dedicated section for Getting started with links to the docs and tutorial, the video would fit perfectly here

Reply:
We can also directly reshare the video on Twitter as part of the Tweets launch campaign

Reply:
[Here's](https://graphcms.com/blog/introducing-configurable-sidebars) an example of what I've done previously with announcement + embedded video.

@Urigo @charlypoly I think we can do a mixture of both. Maybe not for this one (unless we've time) but consider more in-depth videos for announcements. Maybe something along the lines of... 

Post: Here's what it is/how it works
Video: Here's how to build something

Reply:
Agree with you!

We should lay down a format for future announcements because right now we went in 3 directions (not incompatible): long release article with code examples + complete text-tutorial on the website + video tutorial 😅

@notrab I added your video in the dedicated question, so people can choose how to learn Yoga 2:
- Text-tutorial on the website
- your video tutorial
- the docs

Reply:
Should we add the intro video also on Yoga's landing page somewhere?

Reply:
I think it would be great to add it to the documentation index page: https://www.graphql-yoga.com/docs/quick-start

WDYT?